po/es.po

```diff
@@ -7707,7 +7708,7 @@ msgstr "atravesar (monocromo) (experimental)"
 # Demosaicing -> Algoritmo de interpolación cromática.
 #: ../src/iop/demosaic.c:5130 ../src/iop/demosaic.c:5140
 msgid "demosaicing raw data method"
-msgstr "método de interpolación cromática RAW"
+msgstr "Método de interpolación cromática para imágenes en bruto"
```

Comment:
Depending on the context I put one or the other, plus it gives more information to the newbies. I did it according to books, articles, manuals, etc.

Reply:
"imágenes en bruto" doesn't mean anything, only confusion for newbies  because it is not used. Not relevant, but it was the first time I heard that.

If newbies are playing with a RAW processor, don't change the term :)